include/baresip.h

```diff
@@ -1127,7 +1127,13 @@ double video_timestamp_to_seconds(uint64_t timestamp);
 
 const struct rtcp_stats *stream_rtcp_stats(const struct stream *strm);
 struct call *stream_call(const struct stream *strm);
-
+const struct sdp_media *stream_sdp(const struct stream *strm);
+uint32_t metric_get_tx_n_packets(const struct stream *strm);
+uint32_t metric_get_tx_n_bytes(const struct stream *strm);
+uint32_t metric_get_tx_n_err(const struct stream *strm);
+uint32_t metric_get_rx_n_packets(const struct stream *strm);
+uint32_t metric_get_rx_n_bytes(const struct stream *strm);
+uint32_t metric_get_rx_n_err(const struct stream *strm);
```

Comment:
these metric functions should be prefixed with `stream_` which is the same as 
the other functions accessing `struct stream *strm` ..

Reply:
Makes sense.  Once question: Replace metric_ with stream_?  Or simply add stream_ to what's there now?